include/zephyr/net/lwm2m.h

```diff
@@ -1410,7 +1410,23 @@ void lwm2m_rd_client_update(void);
 char *lwm2m_path_log_buf(char *buf, struct lwm2m_obj_path *path);
 
 /** 
- * @brief LwM2M SEND operation to given path list
+ * @brief LwM2M SEND operation to given path list asynchronously with confirmation callback
+ *
+ * @param ctx LwM2M context
+ * @param path_list LwM2M Path string list
+ * @param path_list_size Length of path list. Max size is CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE
+ * @param reply_cb Callback triggered with confirmation state.
```

Comment:
I think the callback should be optional, so this parameter should allow NULL.

Reply:
Precise reply_cb as optionnal parameter (NULL)